### PR TITLE
external link treatment

### LINKED ISSUE #53: Add rel="noreferrer" to external links by default

### PR DESCRIPTION
**Pull request by mbostock**
Fixes #53.

### CODE REVIEW

File: observablehq.config.ts [hunk 1]
```diff
@@ -107,7 +107,7 @@ export default {
   }
 <script type="module">/Win/.test(navigator.platform) || Array.from(document.querySelectorAll(".win"), (e) => e.remove())</script>`,
   header: `<div style="display: flex; align-items: center; gap: 0.5rem; height: 2.2rem; margin: -1.5rem -2rem 2rem -2rem; padding: 0.5rem 2rem; border-bottom: solid 1px var(--theme-foreground-faintest); font: 500 16px var(--sans-serif);">
-  <a href="https://observablehq.com/" style="display: flex; align-items: center;">
+  <a href="https://observablehq.com/" target="self" style="display: flex; align-items: center;">
```
Comment (Fil) [inline observablehq.config.ts:110]: MDN mentions `_self`
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noopener

Comment (mbostock) [reply observablehq.config.ts:110]: Ah, oops, yes, this should be `_self`.

File: src/html.ts [hunk 2]
```diff
@@ -145,6 +145,13 @@ export function rewriteHtml(
     script.setAttribute("src", (isJavaScript(script) ? resolveScript : maybeResolveFile)(src));
   }
 
+  for (const a of document.querySelectorAll<HTMLAnchorElement>("a[href]")) {
+    const href = a.getAttribute("href")!;
+    if (!/^(\w+:)/.test(href)) continue;
+    if (!a.hasAttribute("target")) a.setAttribute("target", "_blank");
```
Comment (Fil) [inline src/html.ts:151]:
```suggestion
```

this is implied by noopener

**Comment by Fil** [comment]
I've added a bit of documentation, and disabled auto-merge to give you a chance to review it.

**Review by mbostock — COMMENTED** [review]
Thanks for adding docs! I expanded them a bit and it looks good to me.

**Pull request merged**